lib/index.coffee

```diff
@@ -51,6 +53,29 @@ streamToString = (stream) ->
 		.on 'end', ->
 			resolve(Buffer.concat(chunks).toString())
 
+setupMetrics = (metrics) ->
+	metrics.describe('gc_space_reclaimed', 'Disk space the GC must free')
+	metrics.describe('gc_number_images_to_remove', 'Number of images to be removed at a given time')
+	metrics.describe('gc_duration_milliseconds', 'Milliseconds taken to free GC the reclaimed space')
+	metrics.describe('gc_image_removal_errors',
```

Comment:
this should be `metrics.describe.counter`, with the "`buckets`" of the config as currently written in fact being `labelNames` (and strings),

```
{
	labelNames: [ "404", "409", "500" ]
}
```

Reply:
:+1:

Reply:
Ah sorry my original comment was odd, you would have it as shown above (edited now). A single label, `statusCode`, and you'd record like:

```coffeescript
@metrics?.inc('gc_image_removal_errors', 1, {statusCode: e.statusCode})
```